kernel/kernel-impl/src/main/resources/antisamy/high-security-policy.xml

```diff
@@ -43,8 +43,8 @@
     <regexp name="htmlTitle" value="[\p{L}\p{N}\s\-_',:\[\]!\./\\\(\)&amp;]*" /> <!-- force non-empty with a '+' at the end instead of '*' -->
     <regexp name="htmlClass" value="[a-zA-Z0-9\s,\-_]+" />
 
-    <regexp name="onsiteURL" value="([\p{L}\p{N}\\\.\#@\$%\+&amp;;\-_~,\?=/!]+|\#(\w)+)" />
-    <regexp name="offsiteURL" value="(\s)*((ht|f)tp(s?)://|mailto:)[\p{L}\p{N}]+[\p{L}\p{N}\p{Zs}\.\#@\$%\+&amp;;:\-_~,\?=/!\(\)]*(\s)*" />
+    <regexp name="onsiteURL" value="([\p{L}\p{N}\\\.\#@\$%\s\+&amp;;\-_~,\?=/!]+|\#(\w)+)" />
```

Comment:
I think `\s` will allow newlines (and others) in URLs, I'm pretty sure this won't be handled well by browsers, is there any reason we need to add more than just a space?